### PR TITLE
[next] Comments Count

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Adds Comment Count

## How do I test this PR?
- Open the Stream
- Check the comment Count
- Write a comment
- Check the comment Count

### CODE REVIEW

**Comment by wyattjoh** [comment]
Could you address the failing tests @okbel?

**Comment by okbel** [comment]
@wyattjoh just pushed the latest snapshots. Should be passing. Will keep an eye on them!

**Review by wyattjoh — APPROVED** [review]

**Comment by alinhle** [comment]
Don't merge this yet, just wanna take a look too on Monday.

**Comment by okbel** [comment]
@cvle We were not going to merge this. We want you to check commit 333e1a6 on Monday since using fragments and queries trigger a weird behaviour with the tests. Have a good weekend!

**Comment by okbel** [comment]
@cvle Tests won't pass for some reason. I updated the fixtures with the extra commentCount, please check and let me know if something is missing.

File: src/core/client/stream/queries/CommentsCountQuery.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        render={({ error, props }) => {
+          if (error) {
+            return <div>{error.message}</div>;
+          }
+
+          if (props && props.asset && props.asset.commentCounts.totalVisible) {
+            return (
+              <CommentCountTab
+                commentCount={props.asset.commentCounts.totalVisible}
+                {...this.props}
```
Comment (alinhle) [inline src/core/client/stream/queries/CommentsCountQuery.tsx]: As talked in meeting: `const {local: _, ...rest} = this.props;`

**Review by alinhle — APPROVED** [review]

**Pull request merged**